daemon/inertiad/build/util.go

```diff
@@ -8,14 +8,17 @@ import (
 	"os"
 	"path/filepath"
 	"strings"
+
+	"github.com/ubclaunchpad/inertia/local"
 )
 
 // getTrueDirectory converts given filepath to host-based filepath if applicable
 // - Docker commands are sent to the mounted Docker socket and hence are
 // executed on the host, using the host's filepaths, which means Docker client
 // commands must use this function when dealing with paths
 func getTrueDirectory(path string) string {
-	return strings.Replace(path, "/app/host", os.Getenv("HOME"), 1)
+	homeDir, _ := local.GetHomePath()
+	return strings.Replace(path, "/app/host", homeDir, 1)
```

Comment:
we should remove this for now - *daemonside* windows compatibility is a much harder and nuanced issue, and a separate ticket is open for that (#407). Also trying to avoid introducing dependencies on `inertia/local` from the daemon for now, to avoid dependency creep 👌

Reply:
Thanks for letting me know. I reverted the changes for util.go.